torch/_torch_docs.py

```diff
@@ -4272,6 +4272,59 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(x, y) -> Tensor
+
+Compute x*log(y) so that the result is 0 if x = 0.
+Similar to SciPy's `scipy.special.xlogy`.
+
+.. math::
+    \text{out}_i = x_i * \log{{y_i}}
```

Comment:
I suggest making this 3 cases (given the current implementation): 

- y = Nan
- x = 0
- general case (this case)